### ISSUE TITLE
Change msgText formatting detection method

### ISSUE THREAD

**Issue by TheFabulousPika** [TheFabulousPika/steam-chatlog-to-text#35]
Consolidate detection algorithm by selecting by class and checking node length
```
function checkFormatting(a,b){
	var thisMsgNode = a;
	var checkClassName = "'[class*=" + b + "]'";
	var testResult = thisMsgNode.querySelectorAll(eval(checkClassName)).length;
  alert(testResult);
	if (testResult == 0){
	return false;
	}
	else {
	return true;
	}
}

**Comment by TheFabulousPika** [comment]
the following functions can be consolidated immediately:
checkIfPre()
checkIfCode()
checkIfQuote()
checkIfFlip()
checkIfStoreLink()

**Comment by TheFabulousPika** [comment]
> the following functions can be consolidated immediately:
> checkIfPre()
> checkIfCode()
> checkIfQuote()
> checkIfFlip()
> checkIfStoreLink()

Completed in #37 

Remaining msgText types
giphy
spoilertext
spoilermedia
graph
me
youtube
img
sticker
trade
tweet
random

**Comment by TheFabulousPika** [comment]
> > the following functions can be consolidated immediately:
> > checkIfPre()
> > checkIfCode()
> > checkIfQuote()
> > checkIfFlip()
> > checkIfStoreLink()
> 
> Completed in #37
> 
> Remaining msgText types
> giphy
> spoilertext
> spoilermedia
> graph
> me
> youtube
> img
> sticker
> trade
> tweet
> random

graph
youtube
img

**Comment by TheFabulousPika** [comment]
> > > the following functions can be consolidated immediately:
> > > checkIfPre()
> > > checkIfCode()
> > > checkIfQuote()
> > > checkIfFlip()
> > > checkIfStoreLink()
> > 
> > 
> > Completed in #37
> > Remaining msgText types
> > giphy
> > spoilertext
> > spoilermedia
> > graph
> > me
> > youtube
> > img
> > sticker
> > trade
> > tweet
> > random
> 
> graph, youtube, img fixed in #40

giphy, voicechat, tradeoffer, tweet, random updated.
me, spoilertext, spoilermedia, sticker do not use a simple className search and thus will be left as-is.
